README.md

```diff
@@ -1,19 +1,20 @@
 # Group 10
 
 ## What you are going to make?
-- GeoBot 	
-- This program will let users control a smart vehicle and possibly turn on automated exploring.  
-- It will be able to create and map an unidentified area.
+- Tank - Total Annihilation Not Komplete
```

Comment:
Should it not be spelled 'Complete' 😆

Reply:
Eyy, I didn't have the heart to change Tank into Tanc you know :P. See the word as a sweedishified version of complete